Translations/fr.po

```diff
@@ -8080,11 +8086,11 @@ msgid ""
 "that. He was an Archbishop, and always seemed to care so much for the "
 "townsfolk of Tristram. So many were injured, I could not save them all..."
 msgstr ""
-"J'ai été choqué quand j'ai entendu parler de ce que les habitants "
-"prévoyaient de faire cette nuit-là. Je pensais que de tous, Lazarus aurait "
-"eu plus de bon sens que ça. Il était archevêque et semblait toujours se "
-"soucier beaucoup des habitants de Tristram. Il y eut tellement de blessés, "
-"je n'ai pas pu tous les sauver..."
+"Je n'en revenais pas quand j'ai appris ce que les habitants se préparaient à "
+"faire cette nuit-là. Je pensais que d'entre tous, Lazarus au moins aurait eu "
```

Comment:
```suggestion
"faire cette nuit-là. Je pensais que d'entre tous, Lazarus aurait eu "
```

Reply:
In this one, I seeked emphasis, illustrating the shock and disappointment, that would be absent without "au moins".